blueocean-rest/src/main/java/io/jenkins/blueocean/rest/model/BluePipelineCreateRequest.java

```diff
@@ -24,19 +23,11 @@ public void setName(String name) {
         this.name = name;
     }
 
-    /* Jenkins organization */
-    public String getOrganization() {
-        return organization;
-    }
-
-    public void setOrganization(String organization) {
-        this.organization = organization;
-    }
-
     /**
      * Create an instance of {@link BluePipeline} from the given request.
      *
      * @return created pipeline
      */
-    public abstract @CheckForNull BluePipeline create(Reachable parent) throws IOException;
+    public abstract @CheckForNull BluePipeline create(Reachable parent, BlueOrganization organization) throws IOException;
```

Comment:
Most places we have Reachable as last parameter and also lets call out organization and parent parameter `@Nonnull`.